Landing/components/eighthSection/styles.ts

```diff
@@ -77,7 +76,10 @@ const fadeIn = keyframes`
 export const CustomField = styled(TextField)`
     background-color: ${({theme})=> theme.palette.background.default} ;
     border-radius: 13px;
+<<<<<<< HEAD
```

Comment:
It will not compile because we have this HEAD

Reply:
done